bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Fan.java

```diff
@@ -40,6 +40,9 @@ static class ChannelConfiguration extends AbstractChannelConfiguration {
 
         @SerializedName("state_topic")
         protected @Nullable String stateTopic;
+        @SerializedName("command_template")
+        @Nullable
+        protected String commandTemplate;
```

Comment:
For "consistency", can you please rather use
`protected @Nullable String commandTemplate;`